addons/payment/models/payment_method.py

```diff
@@ -78,6 +78,11 @@ class PaymentMethod(models.Model):
         required=True,
         default='none',
     )
+    support_manual_capture = fields.Boolean(
```

Comment:
Should this field be a selection with values `full_only` and `partial` so that we can match PMs' behavior with providers?

Reply:
I can change the field to a selection with the following values:
1. full_capture
2. partial_capture
3. no_manual_capture
However, the issue lies in determining the appropriate value for this field, especially for providers like Stripe. They only specify whether they support manual capture or not for a payment method, without clarifying if partial capture is supported. This makes it confusing to set accurate values for this field in the demo data.